SpatialGDK/Source/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -126,7 +125,11 @@ FNetworkGUID FSpatialNetGUIDCache::AssignNewEntityActorNetGUID(AActor* Actor, co
 	UE_LOG(LogSpatialOSPackageMap, Log, TEXT("Registered new object ref for actor: %s. NetGUID: %s, entity ID: %lld"),
 		*Actor->GetName(), *NetGUID.ToString(), EntityId);
 
-	Receiver->ResolvePendingOperations(Actor, ObjectRef);
+	// This will be null when being used in the snapshot generator
```

Comment:
Perhaps use an EDITOR tag instead

Reply:
Do you mean the EDITOR marco? I don't think that's what we want since when we play in editor, that code is still going to be running but the world type has changed.

Reply:
Yep meant the WITH_EDITOR macro. You could wrap just the if loop so we don't have to process this normally.